digging-deeper/artisan-console.md

```diff
@@ -19,7 +19,7 @@ go run . artisan help migrate
 Instead of repeating `go run . artisan ...` command, you may want to add an alias to your shell configuration with the terminal command below:
 
 ```shell
-echo -e "\r\nalias artisan=\"/go run . artisan\"" >>~/.bashrc
```

Comment:
Can you help also change this code in the `zh/digging-deeper/artisan-console.md` file? I added this two days ago. Thanks!